src/Microsoft.AspNet.Server.Kestrel/Http/Connection.cs

```diff
@@ -104,9 +146,9 @@ private void OnRead(UvStreamHandle handle, int status, Exception error)
                 }
             }
 
-
             try
             {
+                ExtractIP();
```

Comment:
Why is ExtractIP called twice?
(I'll still getting familiar with Kestrel's control flow)